### DISCUSSION TITLE
.NET6.0 Target Frameworks C#

### DISCUSSION THREAD

**Discussion by sudharsan-narayanan** [dotnet/wpf#9489] [General]
Hi All,

Can you please explain, why we are using the `TargetFramework` in csproj for net 60 as below?

`<PropertyGroup>
   <TargetFramework>net6.0-windows</TargetFramework>
</PropertyGroup>`

instead of

`<PropertyGroup>
   <TargetFramework>net6.0</TargetFramework>
</PropertyGroup>`

is there any difference in this? Please explain

**Comment by miloush** [comment]
Because WPF works on Windows only. See https://learn.microsoft.com/en-us/dotnet/standard/frameworks for more details.
